cmd/evm/internal/t8ntool/execution.go

```diff
@@ -109,7 +101,13 @@ func (pre *Prestate) Apply(vmConfig vm.Config, chainConfig *params.ChainConfig,
 		receipts    = make(types.Receipts, 0)
 		txIndex     = 0
 	)
-	gaspool.AddGas(pre.Env.GasLimit)
+
+	if chainConfig.IsLondon(new(big.Int).SetUint64(pre.Env.Number)) {
```

Comment:
This is out of date now that we're using `GasLimit` as itself rather than the `GasTarget`.

Reply:
:+1: